SEO optimization for 'aws api gateway'

zchase opened the pull request:
We are currently ranking on the second page for the term "aws api gateway". We are currently ranking 15th and would like to get to the first page for this term and related terms.

<img width="1264" alt="apigatewayseorank" src="https://user-images.githubusercontent.com/15146337/75803870-fac2b100-5d33-11ea-94ae-44a6cb2d32e1.png">

I tried to get the keyword into places that made sense. If you don't think a keyword makes sense were it is, we can remove it.

cnunciato commented on content/docs/guides/crosswalk/aws/api-gateway.md:
```diff
@@ -14,20 +14,20 @@ aliases: ["/docs/reference/crosswalk/aws/api-gateway/"]
-## Defining an API Gateway Endpoint and Routes
-
-API Gateway creates REST APIs that:
+An AWS API Gateway creates REST APIs that:
```
I think this is okay without the additional article, since that's its name.

```suggestion
AWS API Gateway creates REST APIs that:
```

cnunciato commented on content/docs/guides/crosswalk/aws/api-gateway.md:
```diff
@@ -520,27 +520,27 @@ https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-method-
 The `awsx.apigateway.API` class supports three specific methods of controlling access to your APIs:
 
-* *Amazon Cognito user pools* let you create customizable authentication and authorization for your APIs.
+* *AWS Cognito user pools* let you create customizable authentication and authorization for your APIs.
```
I believe for this one, the product name actually is Amazon Cognito, not AWS Cognito.
```suggestion
* *Amazon Cognito user pools* let you create customizable authentication and authorization for your APIs.
```

cnunciato commented on content/docs/guides/crosswalk/aws/api-gateway.md:
```diff
@@ -520,27 +520,27 @@ https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-method-
 
 [Cognito Authorizers](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-integrate-with-cognito.html)
-allow you to use [Amazon Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
+allow you to use [AWS Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
```
```suggestion
allow you to use [Amazon Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
```

cnunciato replied:
```suggestion
allow you to use [Amazon Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
```

cnunciato commented on content/docs/guides/crosswalk/aws/api-gateway.md:
```diff
@@ -14,20 +14,20 @@ aliases: ["/docs/reference/crosswalk/aws/api-gateway/"]
 </a>
 
-[Amazon API Gateway](https://aws.amazon.com/api-gateway/) is a fully managed service for creating, monitoring, and
+Pulumi Crosswalk for AWS provides significantly easier ways of programming an AWS API Gateway. This includes using
```
I actually think this works well, but it's inconsistent with the rest of the pages in this section. So if we do apply this change here, we should probably do so on its siblings as well.

zchase commented:
@cnunciato I took your suggestions and reverted the "What is.." part I added to how it originally was. I think it would be a nice addition SEO wise, but agree that we should be consistent across all pages. Just adding the keywords should be fine for now, and if we feel we still need to optimize the page further we can look at that format change
